src/unimport/utils.py

```diff
@@ -76,12 +74,19 @@ def is_std(package: str) -> bool:
 
 @functools.lru_cache(maxsize=3)
```

Comment:
IMHO you would have a simpler, faster function by moving `("y", "yes", "t", "true", "on", "1")` into a module-level $O(n)$ datastructure such as `frozenset`, and removing this cache